guides/it/docker.md

```diff
@@ -0,0 +1,12 @@
+# Docker
+
+Docker is an open platform for developing, shipping, and running applications. Docker enables developers to separate their applications from their  infrastructure so they can deliver software quickly. With Docker, they can manage their infrastructure in the same ways they manage their applications. By taking advantage of Docker’s methodologies for shipping, testing, and deploying code quickly, they can significantly reduce the delay between writing code and running it in production.
+
+Due to changes within Docker, from Jan 2022 any employees using a Made Tech email address must use the Made Tech licence for Docker Desktop and not an individual licence.
+
+## Onboarding/offboarding
+If you require a Docker licence please message [operations@madetech.com](operations@madetech.com) or slack #ops-it-support with the details of the project / workstream that you need adding to. 
```

Comment:
```suggestion
If you require a Docker licence please message `operations@madetech.com` or slack #ops-it-support with the details of the project / workstream that you need adding to. 
```
Making this a link broke the link checker, I have made a suggestion to change the formating